extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -47,42 +46,22 @@ This extension is required, meaning it must be placed in both the `extensionsUse
-- [Glossary](#glossary)
-- [Extension JSON Schema Reference](#extension-json-schema-reference)
-- [Subtree JSON Schema Reference](#subtree-json-schema-reference)
+- [Multiple contents](#multiple-contents)
 - [Appendix A: Availability Indexing](#appendix-a-availability-indexing)
 
 ## Overview
 
-**Implicit tiling** is a new representation of a Cesium 3D Tileset that allows for fast random access of tiles and enables new traversal algorithms.
-
-Implicit tilesets are uniformly subdivided into a quadtree or octree. This regular pattern allows the tileset to be expressed in a more compact representation which keeps the tileset JSON small.
+This extension defines a concise representation of quadtrees and octrees in 3D Tiles. This regular pattern allows for random access of tiles based on their tile coordinates which enables accelerated spatial queries, new traversal algorithms, and efficient updates of tile content, among other use cases.
 
-The `3DTILES_implicit_tiling` extension can be added to any tile in the tileset. This extension defines how the tile is subdivided and where to locate content resources for the implicit tileset.
+Implicit tiling also allows for better interoperability with existing GIS data formats with implicitly defined tiling schemes. Some examples are [TMS](https://wiki.osgeo.org/wiki/Tile_Map_Service_Specification), [WMTS](https://www.ogc.org/standards/wmts), [S2](http://s2geometry.io/), and [CDB](https://docs.opengeospatial.org/is/15-113r5/15-113r5.html).
 
 In order to support sparse datasets, **availability** data determines which tiles exist. To support massive datasets, availability is partitioned into fixed-size **subtrees**. Subtrees are stored in a compact binary file format.
 
-Implicit tiling allows directly accessing tiles by their **tile coordinates**: `(level, x, y)` for quadtrees or `(level, x, y, z)` for octrees. This avoids traversing the full path to the tile.
-
-<img src="figures/implicit-vs-explicit.jpg" width="600"/>
-
-For a complete list of terminology used, see the [Glossary](#glossary).
-
-## Use Cases
-
-_This section is non-normative_
-
-A key use for implicit tiling is enabling and/or accelerating tree traversal algorithms. Accessing a tile by coordinates is faster than traversing the entire tree. Likewise, raycasting algorithms and GIS algorithms can benefit from the abbreviated tree traversals. Tiles can be loaded immediately instead of going from top to bottom of a tree.
-
-Accessing tiles by coordinates also helps accelerate spatial queries. For example, the highest resolution tile that contains a given point can be quickly located by computing the coordinates of the tile directly. 
+The `3DTILES_implicit_tiling` extension may be added to any tile in the tileset. This extension defines how the tile is subdivided and where to locate content resources. The extension may be added to multiple tiles to create more complex subdivision schemes like double-headed quadtrees.
```

Comment:
"double-headed quadtree" is jargony, if you're going to mention it, explain that it's a quadtree for each hemisphere (e.g. eastern and western), with another root tile that covers the whole world. A diagram would also help. Something like this: https://sandcastle.cesium.com/?src=Imagery%20Layers%20Manipulation.html

![image](https://user-images.githubusercontent.com/8422414/142062758-f0121e92-3be6-44a9-95e3-9b0edb65641d.png)
![image](https://user-images.githubusercontent.com/8422414/142062891-e7b1f283-d7f0-48a6-b48f-75a44abe817c.png)

Except fix the numbering for the eastern hemisphere


Note that I have an example for this: https://github.com/CesiumGS/3d-tiles/issues/512#issuecomment-956636721 albeit this overview section might be too early on to show it.